packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -93,6 +99,19 @@ export default declare((api, options, dirname) => {
       "The 'absoluteRuntime' option must be undefined, a boolean, or a string.",
     );
   }
+  if (
+    typeof relativeRuntime !== "boolean" &&
+    typeof relativeRuntime !== "string"
+  ) {
+    throw new Error(
+      "The 'relativeRuntime' option must be undefined, a boolean, or a string.",
+    );
+  }
+  if (absoluteRuntime !== false && relativeRuntime !== false) {
+    throw new Error(
+      "The 'absoluteRuntime' and 'relativeRuntime' settings cannot be enabled at the same time",
```

Comment:
```suggestion
      "The 'absoluteRuntime' and 'relativeRuntime' options cannot be enabled at the same time",
```

(I think? I'm not a native speaker)

Reply:
Yep, plus it's more in line with the other messages 👍